parser.go

```diff
@@ -285,33 +293,76 @@ func (p *parser) readPythonMultilines(line string, bufferSize int) (string, erro
 	parserBufferPeekResult, _ := p.buf.Peek(bufferSize)
 	peekBuffer := bytes.NewBuffer(parserBufferPeekResult)
 
+	indentSize := 0
 	for {
 		peekData, peekErr := peekBuffer.ReadBytes('\n')
 		if peekErr != nil {
 			if peekErr == io.EOF {
+				if p.options.Debug {
+					fmt.Println("readPythonMultilines: failed to peek")
```

Comment:
Although I understand you would like to have debug info to help debug, but I don't think print to stdout is suitable for this package, i.e. this is not a Go program but a utility package. 

With that being said, I think debug printing should not be committed for this particular package.

Reply:
Alternatively, we can change `Debug bool` to `DebugWriter io.Writer`, and if it's not nil, write debug info to this writer.

Then we just need to think about what to write.

Reply:
In my projects, I use loggers like "github.com/sirupsen/logrus". This allows anyone using your project to configure logging any way they want. not sure if you want to do that here. If so, I can modify the code to use that package

Reply:
Honestly, I don't want to introduce debugging feature and/or another dependency because I'm not convinced this will have enough benefit.

Reply:
OK, I made it a debug function. Will commit changes soon